tsl/test/isolation/specs/compression_ddl.spec

```diff
@@ -11,12 +11,20 @@ setup
     CREATE FUNCTION lock_chunktable( name text) RETURNS void AS $$
     BEGIN EXECUTE format( 'lock table %s IN SHARE MODE', name);
     END; $$ LANGUAGE plpgsql;
+    CREATE FUNCTION count_chunktable(tbl regclass) RETURNS INT[] AS $$
+    DECLARE c int;c_only int;
+    BEGIN
+      EXECUTE format('SELECT count(*) FROM %s', tbl) INTO c;
+      EXECUTE format('SELECT count(*) FROM ONLY %s', tbl) INTO c_only;
+      RETURN ARRAY[c,c_only];
+    END; $$ LANGUAGE plpgsql;
```

Comment:
You can also return a table with column names, which would make it more clear what the numbers represent in the tests.

Reply:
this is intentionally avoided to not expose the chunk names

Reply:
Yea i realized what you meant after i wrote the comment, what i don't like about this is that it makes the calling weirder if you want to benefit from the column names since the function call is in the targetlist and not the FROM-clause, so you need to unpack the TABLE in the targetlist to get the column names.

Reply:
Also this changes the function into SRF and their are additional limitations where srf can be called from.